wikidata_position_history.gemspec

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "wikidata_position_history/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "wikidata_position_history"
+  spec.version       = WikidataPositionHistory::VERSION
+  spec.authors       = ["Tony Bowden"]
```

Comment:
I think you've done enough here that you should either add yourself to this, or explicitly make it the team instead of an individual